hamp_radar/decoders.py

```diff
@@ -187,20 +175,34 @@ def _decode(rawdata):
     return _decode
 
 
-def decode_iq(ppar: Optional[xr.Dataset]):
-    nfft = 256
-    if ppar is not None:
-        nfft = ppar.sft.values  # assumed to be xr.Dataset
+def decode_fftd(rawdata, ppar: Optional[xr.Dataset] = None):
+    if ppar is None:
+        raise ValueError("need PPAR to decode FFTD")
+    # TODO raise eror is osc != [thing]
```

Comment:
Would you implement this todo within this PR?

Reply:
consider it done ;)

Reply:
(also doing radar tag check on decoders too)